element_deeplabcut/train.py

```diff
@@ -249,6 +248,7 @@ class ModelTraining(dj.Computed):
     # https://github.com/DeepLabCut/DeepLabCut/issues/70
 
     def make(self, key):
+        from deeplabcut import train_network

```

Comment:
```suggestion
        from deeplabcut import train_network # isort:skip

```

Reply:
Ok sounds good that makes sense, isort would just move this import back to the top without this comment right?